src/shared/telemetry/activation.ts

```diff
@@ -72,24 +75,28 @@ export async function activate(activateArguments: {
         activateArguments.extensionContext.subscriptions
     )
 }
-
-export function isTelemetryEnabled(toolkitSettings: SettingsConfiguration): boolean {
+/*
+ * Formats the AWS telemetry setting to a boolean: false if setting value was 'Disable' or false, true for everything else
+ */
+export async function formatTelemetrySettingToBool(toolkitSettings: SettingsConfiguration): Promise<void> {
     // Setting used to be an enum, but is now a boolean.
     // We don't have api-based strong type support, so we have to process this value manually.
     const value = toolkitSettings.readSetting<any>(AWS_TELEMETRY_KEY)
 
-    // Handle original opt-out value (setting used to be a tri-state string value)
-    if (value === LEGACY_SETTINGS_TELEMETRY_VALUE_DISABLE) {
-        return false
-    }
+    if (typeof value === 'boolean') return
```

Comment:
Please add braces around all `if` statements even if they're a single line. It's defensive against accidentally leaving stuff out of the statement which leads to unexpected behavior.

We follow this convention everywhere except here apparently 🤔 
https://github.com/aws/aws-toolkit-vscode/blob/41365536e9e580239eeff931f6956a38eb83cd01/src/eventSchemas/commands/downloadSchemaItemCode.ts#L377